starter-code/index.html

```diff
@@ -5,11 +5,26 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Spotify</title>
+    <link rel="stylesheet" href="css/main.css">
   </head>
   <body>
-    Premium Discover Help Download
+    <header>
+      <nav>
+       <img src='/Users/edisonmasis/IronHackLab/lab-css-spotify-clone/starter-code/images/spotify-logo.png'>
+        <div class="nav-wraper">
+         <a href="premium">Premiun</a>
+         <a href="discover">Discover</a> 
+         <a href="help">Help</a> 
+         <a href="download">Download</a>
+       </div>
+     </nav> 
+    </header>  
 
-    Music for everyone.
+    <sectionn class="music-for-everyone">
+      <h1>Music for everyone.</h1>
+    </section>
+
+    
     Spotify is now free on mobile, tablet and computer.
```

Comment:
please try to do more on this lab when you have time and let me know if you need help with it